docs/src/languages/php/extensions.md

```diff
@@ -60,110 +60,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| pdo              | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_dblib        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_firebird     | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_mysql        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_odbc         | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_pgsql        | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_sqlite       | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_sqlsrv       |     |     |     | *   | *   | *   |     | *   |     |
+| http             |     |     | *   |     |     |     |     | *   |     |
+| pgsql            | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| phar             |     |     |     | *   | *   | *   | *   | *   |     |
+| pinba            | *   | *   | *   |     |     |     |     |     |     |
+| posix            |     |     |     | *   | *   | *   | *   | *   |     |
+| propro           |     |     | *   |     |     |     |     |     |     |
+| pspell           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pthreads         |     |     |     |     | *   |     |     |     |     |
+| raphf            |     |     | *   |     |     |     |     | *   | *   |
+| readline         | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| recode           | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| redis            | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| shmop            |     |     |     | *   | *   | *   | *   | *   |     |
+| simplexml        |     |     |     | *   | *   | *   | *   | *   |     |
+| snmp             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| soap             |     |     |     | *   | *   | *   | *   | *   | *   |
+| sockets          |     |     |     | *   | *   | *   | *   | *   |     |
+| sodium           |     |     |     |     |     | *   | *   | *   | *   |
+| sourceguardian   |     |     |     | *   | *   |     |     |     |     |
+| spplus           | *   | *   |     |     |     |     |     |     |     |
+| sqlite3          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| sqlsrv           |     |     |     | *   | *   | *   |     | *   |     |
+| ssh2             | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| sybase           |     |     |     |     | *   | *   | *   | *   | *   |
+| sysvmsg          |     |     |     | *   | *   | *   | *   | *   |     |
+| sysvsem          |     |     |     | *   | *   | *   | *   | *   |     |
+| sysvshm          |     |     |     | *   | *   | *   | *   | *   |     |
+| tideways         |     |     |     | *   | *   | *   | *   | *   |     |
+| tideways-xhprof  |     |     |     | *   | *   | *   | *   | *   |     |
+| tidy             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| tokenizer        |     |     |     | *   | *   | *   | *   | *   |     |
```

Comment:
Why's this one missing?

Reply:
Just missed the buildin 28e56edf24e07dc8fc1e1dc573bd5c38d418b61a